base_layer/core/src/base_node/sync/header_sync/synchronizer.rs

```diff
@@ -781,18 +797,43 @@ impl<'a, B: BlockchainBackend + 'static> HeaderSynchronizer<'a, B> {
     }
 }
 
-struct ChainSplitInfo {
-    local_tip_header: ChainHeader,
-    remote_tip_height: u64,
+#[derive(Debug, Clone)]
+struct PeerChainSplitResponse {
+    block_hashes: Vec<HashOutput>,
     reorg_steps_back: u64,
-    chain_split_hash: HashOutput,
+    headers: Vec<ProtoBlockHeader>,
+    fork_hash_index: u64,
+    remote_tip_height: u64,
+}
+
+/// Information about the chain split from the remote node.
+#[derive(Debug, Clone, PartialEq)]
+pub struct ChainSplitInfo {
+    /// The best block on the local chain.
+    pub best_block: ChainHeader,
```

Comment:
This is badly named. Is it the best header we have or the best block we have?

I made a few comments already, but looks like it could just be that this should have been `best_header`

Reply:
This is exactly the fix we made, its the header of the best block. 
Best block and best header are not always the same

Reply:
Fixed te name